test/promql/tests/example.yaml

```diff
@@ -0,0 +1,21 @@
+evaluation_interval: 30s
```

Comment:
Are these changes related to the extraction of the Prometheus rules ?

Reply:
Yes. We introduce a rest here that points to the extracted rule file. This proves the process works.